### DISCUSSION TITLE
Prompt Copilot

### DISCUSSION THREAD

**Discussion by Sidak08** [zed-industries/zed#8386] [Feature Requests]
How do I Prompt Copilot in Zed. I am logged in and it is making suggestions. pressing cmd + i brings up open ai.

**Comment by SomeoneToIgnore** [comment]
There's `zed: open default keymap` command that shows this for me:

```
"alt-\\": "copilot::Suggest",
"alt-]": "copilot::NextSuggestion",
"alt-[": "copilot::PreviousSuggestion",
```

note that `\\` is an escape of `\`.
Seem to work for me, but there could be caveats like `something.<caret>)` where it might not appear despite asking for the suggestion.

**Reply by Sidak08** [reply]
I was asking about how I can bring up the prompt window like you can in vs code by pressing cmd + i.

**Reply by SomeoneToIgnore** [reply]
Nothing like that exists in Zed at the moment then, current functionality only has a way to provide the completion, accept/decline one or ask for another one.
